File: sdkwork-backend-sdk-typescript/src/types/page-plus-visit-history-vo.ts
import type { PageableObject } from './pageable-object';
import type { PlusVisitHistoryVO } from './plus-visit-history-vo';
import type { SortObject } from './sort-object';

export interface PagePlusVisitHistoryVO {
  totalPages?: number;
  totalElements?: number;
  size?: number;
  content?: PlusVisitHistoryVO[];
  number?: number;
  numberOfElements?: number;
  pageable?: PageableObject;
  sort?: SortObject;
  first?: boolean;
  last?: boolean;
  empty?: boolean;
}

File: sdkwork-backend-sdk-go/types/page_plus_visit_history_vo.go
package types


type PagePlusVisitHistoryVO struct {
	TotalElements int `json:"totalElements"`
	TotalPages int `json:"totalPages"`
	Size int `json:"size"`
	Content []PlusVisitHistoryVO `json:"content"`
	Number int `json:"number"`
	NumberOfElements int `json:"numberOfElements"`
	Sort SortObject `json:"sort"`
	Pageable PageableObject `json:"pageable"`
	First bool `json:"first"`
	Last bool `json:"last"`
	Empty bool `json:"empty"`
}
